state/disabled/disabledTrieLeafParser.go

```diff
@@ -0,0 +1,24 @@
+package disabled
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/core"
+	"github.com/ElrondNetwork/elrond-go-core/core/keyValStorage"
+)
+
+type disabledTrieLeafParser struct {
```

Comment:
maybe rename this to `trieLeafParserV1` ? 
As this will be the first version we have implemented

And the package should be `parsers` ?

Reply:
Refactored.